npm/pkg/dataplane/policies/policy_windows.go

```diff
@@ -57,6 +57,20 @@ func (orig NPMACLPolSettings) compare(newACL *NPMACLPolSettings) bool {
 		orig.Priority == newACL.Priority
 }
 
+func (networkPolicy *NPMNetworkPolicy) numACLRulesProducedInKernel() int {
+	// NOTE: this could be optimized with just len(policy.ACLs) if we don't use the "Both" Direction
+	numRules := 0
```

Comment:
Even in Windows we have the extra block rule we should calculate. So the linux function is actually good for both OS.